### PR TITLE
Added "_" sufix to internal variables

### PR DESCRIPTION
**Pull request by viquezclaudio**
# 🚀 Description

Added a _ suffix to all internal state variables

- [ x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [ ] ✅ I've added tests where applicable to test my new functionality.
- [ ] 📖 I've made sure that my contracts are well-documented.
- [ ] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

### CODE REVIEW

**Comment by viquezclaudio** [comment]
Greetings, This is a WIP PR

**Comment by elopio** [comment]
Hello @viquezclaudio! It's so good to see you here :)
Sorry, I should have been clearer specifying this change to you.
We only want the suffix underscore on internal and private state variables. For example, we want it in:

```
RefundEscrow private escrow_;
```

But not on:

```
address researcher_ = researchers[_target];
```

I also don't think we need the suffix underscore on struct fields.

**Comment by viquezclaudio** [comment]
@elopio OK, will do that. That's why I asked if by internal state variables we mean "any variable that has no modifier or is private". Because under that category the for loops iterators can be included, and that's probably not what you want

**Comment by elopio** [comment]
Nop, loops iterators don't need underscore either.

**Comment by viquezclaudio** [comment]
@elopio I just applied your comments

File: contracts/mocks/BurnableTokenMock.sol [hunk 1]
```diff
@@ -6,7 +6,7 @@ import "../token/ERC20/BurnableToken.sol";
 contract BurnableTokenMock is BurnableToken {
 
   constructor(address _initialAccount, uint256 _initialBalance) public {
-    balances[_initialAccount] = _initialBalance;
+    balances_[_initialAccount] = _initialBalance;
```
[the remark this answers is not in the document]
Comment (viquezclaudio) [reply contracts/mocks/BurnableTokenMock.sol]: I agree

**Comment by viquezclaudio** [comment]
@elopio I applied your comments, and documented on CODE_STYLE this new rule. Let me know if something else is needed.

**Comment by elopio** [comment]
@viquezclaudio because it was my fault that this got delayed and got conflicts with master, I've resolved them and pushed to your branch. I hope you don't mind. I also made a little tweak on CODE_STYLE. I'm +1 this, but it needs another review from somebody who hasn't touched the code. @frangio @nventuro @shrugs, can one of you please take a look?

File: contracts/mocks/ERC721ReceiverMock.sol [hunk 2]
```diff
@@ -16,8 +16,8 @@ contract ERC721ReceiverMock is ERC721Receiver {
   );
 
   constructor(bytes4 _retval, bool _reverts) public {
-    retval = _retval;
-    reverts = _reverts;
+    retval_ = _retval;
```
Comment (nventuro) [inline contracts/mocks/ERC721ReceiverMock.sol]: Reading this made me die a little on the inside :cry:

File: contracts/mocks/ERC721ReceiverMock.sol [hunk 3]
```diff
@@ -4,8 +4,8 @@ import "../token/ERC721/ERC721Receiver.sol";
 
 
 contract ERC721ReceiverMock is ERC721Receiver {
-  bytes4 retval;
-  bool reverts;
+  bytes4 retval_;
```
Comment (nventuro) [inline contracts/mocks/ERC721ReceiverMock.sol]: Can we make the visibility explicit here (and add a Solium rule if possible)?

Comment (elopio) [reply contracts/mocks/ERC721ReceiverMock.sol]: That should be a separate issue:
https://github.com/OpenZeppelin/openzeppelin-solidity/issues/1250

**Comment by elopio** [comment]
@nventuro: @viquezclaudio is already working on the solium rule: https://github.com/OpenZeppelin/solium-plugin-zeppelin/pull/31

**Pull request merged**